src/helix-ui/elements/_hx-tooltip.less

```diff
@@ -0,0 +1,38 @@
+@import './_hx-element';
+@import '../styles/modules/mix-position.less';
```

Comment:
From the PR of the big refactor this probably needs to be split up but wanted to get this in then can pull out the `:host` styles